README.md

```diff
@@ -84,7 +84,6 @@ Name | Website | Region
 [BetterUp](/company-profiles/betterup.md) | https://www.betterup.com/ | USA
 [BeyondPricing](/company-profiles/beyondpricing.md) | https://beyondpricing.com | USA, Europe
 [Big Cartel](/company-profiles/big-cartel.md) | https://www.bigcartel.com | USA
-[Big Wheel Brigade](/company-profiles/big-wheel-brigade.md) | https://www.bigwheelbrigade.com/ | USA
```

Comment:
Please also remove the company profile in the company-profiles folder.

Reply:
sure, one moment.

Reply:
Thank you!